packages/vats/scripts/revive-kread.js

```diff
@@ -0,0 +1,13 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').ProposalBuilder} */
+export const defaultProposalBuilder = async () =>
+  harden({
+    sourceSpec: '@agoric/vats/src/proposals/kread-proposal.js',
+    getManifestCall: ['getManifestForKread', harden({})],
```

Comment:
Unnecessary nested `harden`.
```suggestion
    getManifestCall: ['getManifestForKread', {}],
```
But for that matter, this argument is ignored by packages/vats/src/proposals/kread-proposal.js `getManifestForKread` anyway and should itself be unnecessary.
```suggestion
    getManifestCall: ['getManifestForKread'],
```